File: go/fib.go
package main

import (
	"fmt"
	"time"
)

func fib(n int) int {
	if (n < 2) { return n }
	return fib(n - 2) + fib(n - 1)
}

func main() {
	fmt.Println("Start warming up.")
	sum := fib(38)
	fmt.Println("Finish warming up.")

	start := time.Now()
	sum += fib(38)
	duration := time.Now().Sub(start)

	fmt.Printf("Duration: %f\n", duration.Seconds())
	fmt.Printf("Result: %d\n", sum)
}

File: nodejs/fib.js
function fib(n) {
    if (n < 2) return n;
    return fib(n - 2) + fib(n - 1);
}

exports.handler = function(event, context) {
    console.log('Start warming up.');
	var sum = fib(38);
    console.log('Finish warming up.');

	var start = new Date();
	sum += fib(38);
	var duration = (new Date() - start) / 1000;

	console.log('Duration: ', duration);
	console.log('Result: ', sum);

	context.succeed("OK");
};